javascript/javascript1/week3/savenote.js

```diff
@@ -0,0 +1,37 @@
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({
```

Comment:
two comments:

1. you do not check for non-number values in your save function. This allows you to save non-number ids, but not retrieving later on. This would cause a bug in a real-life scenario where certain values are saved with ids that can then not be accessed.

2. ids are by definition unique. what happens if i save 2 entries with the same id? This would again cause a bug when invoking the *getter* function. Can we prevent this from occuring?

Reply:
Got it. Changed now.